src/api/Jalasoft.TeamUp.Projects.API/Controllers/PatchInvitationById.cs

```diff
@@ -0,0 +1,69 @@
+namespace Jalasoft.TeamUp.Projects.API.Controllers
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.IO;
+    using System.Net;
+    using System.Threading.Tasks;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Jalasoft.TeamUp.Projects.ProjectsException;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.JsonPatch;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.OpenApi.Models;
+    using Newtonsoft.Json;
+
+    public class PatchInvitationById
+    {
+        private readonly IInvitationsService updateInvitationService;
+
+        public PatchInvitationById(IInvitationsService updateInvitationService)
+        {
+            this.updateInvitationService = updateInvitationService;
+        }
+
+        [FunctionName("UpdateInvitation")]
+        [OpenApiOperation(operationId: "UpdateInvitation", tags: new[] { "Invitations" })]
+        [OpenApiParameter(name: "id", In = ParameterLocation.Path, Required = true, Type = typeof(Guid), Description = "The invitation identifier.")]
+        [OpenApiRequestBody("application/json", typeof(JsonPatchDocument<Invitation>), Description = "JSON request body")]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Invitation), Description = "Successful response")]
+        [OpenApiResponseWithoutBody(statusCode: HttpStatusCode.NotFound, Description = "Resource not found")]
+        public async Task<IActionResult> UpdateInvitation(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "patch", Route = "v1/invitations/{id}")] HttpRequest req, Guid id)
+        {
+            try
+            {
+                var invitation = this.updateInvitationService.GetInvitation(id);
```

Comment:
since you return an invitation from the invitations service you could validate that object to determine the existence of the invitation, that way you don't need to get the invitation